src/main/java/games/strategy/triplea/printgenerator/CountryChart.java

```diff
@@ -23,12 +23,11 @@
   private final Map<Territory, List<Map<UnitType, Integer>>> infoMap = new HashMap<>();
 
   protected void saveToFile(final PlayerID player, final PrintGenerationData printData) {
-    final GameData m_data = printData.getData();
-    final PrintGenerationData m_printData = printData;
```

Comment:
Both `m_printData` and `printData` refer to the same instance, so I simply removed the `m_printData` local variable and replaced all occurrences in scope with `printData`.

You'll see this change in pretty much every file from the `printgenerator` package, so this comment applies to all locations.